app/src/ActionModal/components/DelegationModal.vue

```diff
@@ -325,6 +324,16 @@ export default {
     enhancedTargetValidator() {
       return validatorEntry(this.targetValidator)
     },
+    smallestAmount() {
+      if (
+        this.currentNetwork.network_type === "polkadot" &&
+        ["controller", "stash/controller"].includes(this.session.addressRole)
+      ) {
+        return 0
```

Comment:
I understand the idea here, but I actually think we will just not show the amount field when the goal is just to select/nominate a validator.

The idea actually you can see it in my PR #4783. Instead of `Stake`, the button on PageValidator will say `Select` and won't display the amount field. For staking users will have the `Stake` button on Portfolio

Reply:
Great! definitely better ux! We can merge this meanwhile your PR is ready (or happy to delete if it's inminent).

Reply:
I think the merge is quite imminent, or so it seems